docs/api-reference/catalogd-webserver.md

```diff
@@ -13,6 +18,21 @@ As an example, to access the full FBC via the v1 API endpoint (indicated by path
 
 the URL to access the service would be `https://catalogd-service.olmv1-system.svc/catalogs/operatorhubio/api/v1/all`
 
+2. `api/v1/metas` endpoint that allows clients to retrive filtered portions of the FBC. 
+
+The metas endpoint accepts additional parameters following the pattern `/api/v1/metas?<parameters>`, and accepts parameters that corresponds to any combination of the fields of the [FBC Meta structure](https://olm.operatorframework.io/docs/reference/file-based-catalogs/#schema) (except for `Blob`).
+
+As an example, to access only the [package schema](https://olm.operatorframework.io/docs/reference/file-based-catalogs/#olmpackage-1) blobs of the FBC via the `api/v1/metas` endpoint where `.status.urls.base` is
+
+```yaml
+    urls:
+        base: https://catalogd-service.olmv1-system.svc/catalogs/operatorhubio
+```
+
+the URL to access the service would be `https://catalogd-service.olmv1-system.svc/catalogs/operatorhubio/api/v1/metas?schema=olm.package`
+
+For more examples of valid queries that can be made to the `api/v1/metas` service endpoint, please see [Catalog queries](../howto/catalog-queries.md).
```

Comment:
Should "Queries" be capitalized, or `catalog` be lower-case? The link text seems inconsistent.